custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py

```diff
@@ -75,6 +77,11 @@ def generate_csv(self):
                     '{}_site_code'.format(loc_type): current_location['site_code'],
                     '{}_is_test'.format(loc_type): 1 if metadata.get('is_test_location') == 'test' else 0,
                 })
+                if loc_type == 'supervisor':
+                    loc.update({
+                        'ls_deprecates': ','.join(metadata.get('deprecates', [])),
```

Comment:
why the variable names are ls_deprecates not supervisor_deprecates which matches to the existing naming?

Reply:
No reason. I can change that